test/parallel/test-blob-buffer-too-large.js

```diff
@@ -0,0 +1,19 @@
+// Flags: --no-warnings
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const { Blob } = require('buffer');
+
```

Comment:
Looks like this test makes the runner crash, a quick fix would be to skip this test on that platform.
```suggestion

if (common.isFreeBSD) common.skip('Oversized buffer make the FreeBSD CI runner crash');

```

Reply:
Thanks for your insight!
I try to skip this test for freeBSD.